src/ConnectedMode/Embedded/SonarLintTargets.xml

```diff
@@ -20,11 +20,14 @@
 
     <PropertyGroup Condition="$(BuildingInsideVisualStudio) == 'true'">
         <SolutionDirAsInVs>$([System.IO.Path]::GetDirectoryName($(SolutionDir)))</SolutionDirAsInVs>
+        <SonarLanguage>$(Language)</SonarLanguage>
+        <SonarLanguage Condition=" $(Language) == 'C#' OR $(Language) == 'c#'">csharp</SonarLanguage>
+        <SonarLanguage Condition=" $(Language) == 'VISUALBASIC'">vb</SonarLanguage>
```

Comment:
According to this website, apparently those are the possible values, I don't know if there is a better/official documentation: https://documentation.help/MSBuild/P_MSBuild_Community_Tasks_Script_Language.html

Reply:
`#` is a valid character in folder and file names on Windows so in theory we could just use `$(Language)` directly, although it does feel a bit weird. What do you think?